### ISSUE TITLE
Games crash w/ driver installed

### ISSUE THREAD

**Issue by TheFlano23** [timschumi/iforce-binary-driver#13]
Just installed this driver by disabling sign enforcement and upon launching into either ETS2 or Assetto Corsa, the rightmost LED of my WingMan Formula Force blinks and the results differ from there (with ETS2 the game outright crashes (crash log included), with Assetto Corsa it returns to main menu as soon as I start a race). Only just received this steering wheel today so there could be some big step I'm missing, either way any help would be much appreciated

[game.crash.txt](https://github.com/user-attachments/files/16925495/game.crash.txt)

**Comment by TheFlano23** [comment]
For anyone else who may be experiencing this problem, I was able to find a partial workaround for Assetto Corsa by going to game settings and forcing 32 bit before starting. The framerate is not good by any means but I'm able to use the wheel and play the game to some extent.

**Comment by TheFlano23** [comment]
Was finally able to fix the problem entirely with some more digging, my variation is exclusively for the Formula Force but if you know what you're doing you can alter the PowerShell script to include your wheel. 

1. Go to [https://github.com/WallyCZ/saitek-cyborg-ff](this URL) and download / extract it to a folder.
2. Edit the .ps1 file and look for $targets
3.  Type in something like this (not sure if hardware ID's differ by client but you can find yours by going into Device Manager - WingMan Formula Force - Properties - Details - Hardware IDs): 
`    @{ DllName = "SaiQFF04.dll"; DeviceName = "WingMan Formula Force"; HardwareId = "VID_046D&PID_C291"; RenamedDevice = "Logitech WingMan Formula Force (USB)" }` 
4. Upon running the file (with admin) it'll ask you if you want to rename the device, say yes

After that I was able to play 64 bit versions of both ETS2 and AC, force feedback on both and well over 60fps. Hoping this helps someone

**Comment by timschumi** [comment]
Thanks for looking into it and validating that the fix also applies to this driver.

Just for posterity, that means that this is essentially a duplicate of #9, which was already somewhere on my table.
